client/app/components/packages/rwcds-form/proposed-project-actions-editor.js

```diff
@@ -1,4 +1,5 @@
 import Component from '@glimmer/component';
+import { AFFECTED_ZONING_RESOLUTION_ACTION_OPTIONSET } from '../../../models/affected-zoning-resolution';
```

Comment:
Can import the optionset helper here, intead:

```
import { optionset } from '../../../helpers/optionset';
```

Refer to comment below for reason behind this change